docs/ballerina-by-example/examples/websocket-chat-application/websocket-chat-application.bal

```diff
@@ -1,27 +1,34 @@
 import ballerina.net.ws;
 
 @ws:configuration {
-    basePath: "/chat/{fname}+{lname}/{age}",
+    basePath: "/chat/{name}",
     port:9090
 }
 service<ws> ChatApp {
 
     map consMap = {};
 
-    resource onOpen(ws:Connection conn, string fname, string lname, string age) {
+    resource onOpen(ws:Connection conn, string name, string age) {
```

Comment:
Only the path params should be resource arguments. Since "age" is not a path param can we remove it?